doc/techref/fonts.md

```diff
@@ -0,0 +1,36 @@
+# Supported Fonts
+
+PyGMT supports the 35 standard PostScript fonts. The table below lists the 35 fonts with
+font number and the font name.
```

Comment:
I am wondering if it makes sense to have all the texts parts directly at the beginning:

![fonts_text_together](https://github.com/user-attachments/assets/072e891f-836c-4ab1-9faa-601b79255eb4)

```suggestion
PyGMT supports the 35 standard PostScript fonts. The table below lists them with their
font numbers and font names. When specifying fonts in PyGMT, you can either give the 
entire font name or just the font number. For example, to use the font "Helvetica", you
can use either `"Helvetica"` or `"0"`.
For the special fonts "Symbol" (12) and "ZapfDingbats" (34), see the {doc}`/techref/encodings`
for the character set.
The image below the table shows a visual sample for each font.
```